frontend/src/constants/CreateAssessmentConstants.ts

```diff
@@ -1,9 +1,51 @@
 import { Grade } from "../APIClients/types/UserClientTypes";
+import { AssessmentProperties } from "../types/AssessmentTypes";
+import { StringOption } from "../types/SelectInputTypes";
 import { removeUnderscore, titleCase } from "../utils/GeneralUtils";
 
-const gradeOptions = Object.values(Grade).map((grade) => ({
+export const gradeOptions = Object.values(Grade).map((grade) => ({
   value: grade,
-  label: grade === "K" ? "K" : titleCase(removeUnderscore(grade)),
+  label: titleCase(removeUnderscore(grade)),
 }));
 
-export default gradeOptions;
+export interface TestFilterOptions {
+  gradeOptions: StringOption[];
+  testTypeOptions: StringOption[];
+  countryOptions: StringOption[];
+  regionOptions: StringOption[];
+}
+
+export const getTestFilterOptions = (
```

Comment:
could we move this function from `constants` to `utils`? may need to create the appropriate file. honestly could probably also move `gradeOptions` there too (& then get rid of the file?)

also thoughts on renaming to `testFilterOptions` to be consistent with `gradeOptions`? (or can change to `getGradeOptions`, just wanna be consistent!

alsoooooo we should probably be consistent about either using `test` or `assessment`. i think for now, we've been using `assessment` on the frontend and `test` on the backend, but the lines are starting to blur haha